### PR TITLE
Improving css selectors

### PR DESCRIPTION
**Pull request by veronicatofino**

### CODE REVIEW

File: src/page/address-step.page.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ export class AddressStepPage {
   private nextStep: ElementFinder;
 
   constructor () {
-    this.nextStep = $('#center_column > form > p > button > span');
+    this.nextStep = $('.cart_navigation > button');
```
Comment (manuelq12) [inline src/page/address-step.page.ts]: You can improve more this selector

File: src/page/product-added-modal.page.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ export class ProductAddedPage {
   private summaryPage: ElementFinder;
 
   constructor () {
-    this.summaryPage = $('[style*="display: block;"] .button-container > a');
+    this.summaryPage = $('#layer_cart .button-container > a');
```
Comment (manuelq12) [inline src/page/product-added-modal.page.ts]: You can improve more this selector

File: src/page/shipping-step.page.ts [hunk 3]
```diff
@@ -6,7 +6,7 @@ export class ShippingStepPage {
 
   constructor () {
     this.serviceTerms = $('#cgv');
-    this.nextStep = $('#form > p > button > span');
+    this.nextStep = $('.cart_navigation > button');
```
Comment (manuelq12) [inline src/page/shipping-step.page.ts]: You could improve  more this selector

File: src/page/summary-step.page.ts [hunk 4]
```diff
@@ -4,7 +4,7 @@ export class SummaryStepPage {
   private nextStep: ElementFinder;
 
   constructor () {
-    this.nextStep = $('.cart_navigation span');
+    this.nextStep = $('.cart_navigation > a.button');
```
Comment (manuelq12) [inline src/page/summary-step.page.ts:7]: Why is this one a better selector?

**Review by yesid-lopez — APPROVED** [review]

File: src/page/summary-step.page.ts (hunk 4 above)
Comment (veronicatofino) [reply src/page/summary-step.page.ts:7]: I didn't see it as a better selector. At first I tried to homogenize all the buttons of "Proceed to checkout" with the same selector, so I changed it, but as most buttons have names and it is a better solution as a selector, the change is no longer so necessary.

**Review by manuelq12 — APPROVED** [review]

**Pull request merged**